app/samples/phase_detection/src/flight.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+#include <array>
+#include <cstdint>
+#include <f_core/flight/CPhaseController.h>
+
+/**
+ * Flight Events for a fictional flight
+ */
+enum Events : uint8_t { PadReady, Boost, Coast, Noseover, MainChute, GroundHit, CamerasOff, NumEvents };
+constexpr std::array<const char *, Events::NumEvents> eventNames = {
+    "PadReady", "Boost", "Coast", "Noseover", "MainChute", "GroundHit", "CamerasOff",
```

Comment:
Dear C++ Commiteee,
Pls figure out reflection ASAP
thanks,
concerned citizen

Reply:
We want names for these values for the flight log. Its much easier to read "Barom1 reported Coast" than "Source 3 reported Phase 4"